seqr/views/apis/staff_api_tests.py

```diff
@@ -380,3 +640,31 @@ def test_success_story(self):
 
         self.assertEqual(len(response_json['rows']), 1)
         self.assertDictEqual(response_json['rows'][0], EXPECTED_SUCCESS_STORY)
+
+    @mock.patch('seqr.views.apis.staff_api.export_multiple_files')
+    @responses.activate
+    def test_anvil_export(self, mock_export_multiple_files):
+        url = reverse(anvil_export, args=[PROJECT_GUID])
+        _check_login(self, url)
+
+        # Create a SavedVariant row needed by this test
+        saved_variant = SavedVariant.objects.get(guid__exact = "SV0000001_2103343353_r0390_100")
+        variant_tag_type = VariantTagType.objects.get(name__exact = "Known gene for phenotype")
+        variant_tag = VariantTag.objects.create(variant_tag_type = variant_tag_type)
+        variant_tag.saved_variants.add(saved_variant)
+        variant_tag.save()
```

Comment:
You should put this in the fixture data, not have it dynamically created here

Reply:
I did put it in the fixture data. But it breaks some save_variant_api tests and some other tests. So I decided not to change the fixture data. If it must be in the fixture data, then I can find out the issue and make a new data which won't break other tests.

Reply:
This should be fixture data. You may need to update other tests, thats okay

Reply:
Fixed.